### PR TITLE
[review] Add git specific stuff!

### PR DESCRIPTION
**Pull request by dequis**
So let's start using pull requests for official reviews by testing something that involves almost no code at all!

This covers most of the differences I had between the "master" and "develop" branches of my fork.

Mostly looking for comments on the readme wording, and well, wilmer told me he doesn't like how README.md is so github specific and web 2.0 soooo we don't really agree there. Could be turned into plaintext, maybe, but this is fancier and I like fancy stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+See ./doc/README and ./doc/HACKING for more details.
+
+## Help?
+
+Join **#BitlBee** on OFTC (**irc.oftc.net**) (OFTC, *not* FreeNode!) and flame us right in the face. :-)
+
+
+[html]: http://bzr.dequis.org/revmap/fancy.html
```
Comment (jelmer) [inline README.md]: These three links seem irrelevant now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Wiki: http://wiki.bitlbee.org/
+
+License: GPLv2
+
+## Development
+
+Use github pull requests to submit patches.
+
+Mappings of bzr revisions to git commits: [html][], [raw][] (created using [git-bzr-rev-map][])
```
Comment (jelmer) [inline README.md]: this is now irrelevant.

File: doc/Makefile [hunk 3]
```diff
@@ -5,7 +5,7 @@ endif
 
 all:
 	# Only build the docs if this is a bzr checkout
```
Comment (jelmer) [inline doc/Makefile]: s/bzr checkout/git tree/

File: README.md (hunk 2 above)
Comment (dequis) [inline README.md]: Actually, I just updated it today to cover up to 1072. It's mostly there for historical purposes, such as old comments referencing bzr revisions

Comment (jelmer) [inline README.md]: I think you'd want to check these files in; having them hosted externally seems like a good way to lose that information in the future.

Comment (dequis) [inline README.md]: Good point. Can it be just the "raw" one? (the one that's a tab separated text file). They are kinda big and the fancy one is a bit redundant.

```
232K    raw.txt
736K    fancy.html
```

And filename `doc/git-bzr-rev-map`?

Comment (jelmer) [inline README.md]: seems reasonable to me :)

**Comment by dequis** [comment]
Pushed some stuff, to squash later. Added .travis.yml but it's not enabled for this repo on their side yet.

File: Makefile [hunk 4]
```diff
@@ -152,7 +152,7 @@ tar:
 	fakeroot debian/rules clean || make distclean
 	x=$$(basename $$(pwd)); \
 	cd ..; \
-	tar czf $$x.tar.gz --exclude=debian --exclude=.bzr* --exclude=.depend $$x
+	tar czf $$x.tar.gz --exclude=debian --exclude-vcs --exclude-vcs-ignores $$x
```
Comment (jgeboski) [inline Makefile]: The `--exclude-vcs` and `--exclude-vcs-ignores` are only in newer tar versions. Might want to research this a bit more.

**Comment by dequis** [comment]
Squased [this tree][1] into e26aa72. Looks good?

[1]: https://github.com/bitlbee/bitlbee/commits/4d993e219f76fcefeccceb43d9f62b897330ddfc

**Comment by jgeboski** [comment]
Looks good.
